src/plugins/inspector/public/views/requests/components/details/req_code_viewer.tsx

```diff
@@ -119,37 +80,13 @@ export const RequestCodeViewer = ({
               </EuiCopy>
             </div>
           </EuiFlexItem>
-          {shouldShowDevToolsLink && (
-            <EuiFlexItem grow={false}>
-              <div>
-                <EuiButtonEmpty
-                  size="xs"
-                  flush="right"
-                  iconType="wrench"
-                  href={consoleHref}
-                  onClick={handleDevToolsLinkClick}
-                  data-test-subj="inspectorRequestOpenInConsoleButton"
-                >
-                  {openInConsoleLabel}
-                </EuiButtonEmpty>
-              </div>
-            </EuiFlexItem>
-          )}
-          {shouldShowsearchProfilerLink && (
-            <EuiFlexItem grow={false}>
-              <div>
-                <EuiButtonEmpty
-                  size="xs"
-                  flush="right"
-                  iconType="visBarHorizontal"
-                  href={searchProfilerHref}
-                  onClick={handleSearchProfilerLinkClick}
-                  data-test-subj="inspectorRequestOpenInSearchProfilerButton"
-                >
-                  {openInSearchProfilerLabel}
-                </EuiButtonEmpty>
-              </div>
-            </EuiFlexItem>
+          {includeOpenInAppLinks && (
```

Comment:
Then you can just render like
```
actions.map((name, action) => (
    <EuiFlexItem grow={false} key={name}>
         {action}
    </EuiFlexItem>
))
```

That way, action doesn't need to be wrapped in `EuiFlexItem` and leaking how these are consumed into the the actions themselves.